### PR TITLE
Catalog integrated with map

### PR DESCRIPTION
**Pull request by tjncgit**
- added code for catalog to usermap.js
- just need to provide place id to getCatalog function

### CODE REVIEW

File: project/src/main/webapp/usermap.js [hunk 1]
```diff
@@ -117,6 +117,73 @@ function radiusChange(sel) {
...
+function next() {
+    i += 1;
+    if (i < photoURLS.length) {
+        image = document.getElementById("image");
+        image.setAttribute("src", photoURLS[i]);
+    }
+}
+
+function previous() {
+    i -= 1;
```
Comment (Howon) [inline project/src/main/webapp/usermap.js]: This is an essential duplicate of what you have in usermaps.js. Can we try to dedup these somehow?

**Review by Howon — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/usermap.js (hunk 1 above)
Comment (tjncgit) [reply project/src/main/webapp/usermap.js]: confused about this comment

Comment (Howon) [reply project/src/main/webapp/usermap.js]: Right now you have this global function `previous` and `next` in usermap.js. you also have the same funcitons in catalog.js.

maybe you can make a Pagination class that encapsulates these funcitons?

class Pagination {
  next()
  previous()
}

**Review by Howon — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/usermap.js (hunk 1 above)
Comment (tjncgit) [reply project/src/main/webapp/usermap.js]: okay, didnt meant to have does functions in 2 different files only in the catalog.js

**Pull request closed without merging**